packages/icons/package.json

```diff
@@ -3,28 +3,30 @@
   "version": "0.1.1",
   "description": "Icons for Cockroach UI exported as React Components",
   "files": [
-    "dist/index.js",
-    "dist/index.d.ts"
+    "dist/"
   ],
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
-    "build": "webpack",
-    "build:watch": "webpack --watch",
-    "clean": "rimraf dist",
+    "build": "npm-run-all build:generate build:typescript",
+    "build:typescript": "tsc",
+    "build:generate": "svgr --typescript --icon -d src/components svg",
```

Comment:
Wondering if it would be more clear if the generated components were added to `dist` instead of `src`. Is there a reason they are added to `src`?